### ISSUE TITLE
Workers have gone online

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/SilentCryptoMiner#2054]
Hello Unam.  I have a serious problem.  All workers have gone offline.  There were over 100 online every day on miningocean and 2miners.  Yesterday everything was fine, but today I discovered this problem.

**Comment by VK-VZ** [comment]
I found the problem.  I was hacked.  There was no entry to Vds.  This means it was the unam panel that was hacked.  Is there any way to make the panel more secure?  Maybe set 2fa to login.

**Comment by VK-VZ** [comment]
Yes you are right.  I found my password in the php error files.  What should I do in this situation?  After all, the one who hacked me will do it again and every day.

**Comment by VK-VZ** [comment]
Maybe you can close this section from viewing?

**Comment by UnamSanctam** [comment]
Yes either you can forbid the directory from being viewed, or you can fix what the issue is that causes your error (for post it here so I can) or you can disable the error log in the files by putting a `//` before this line: https://github.com/UnamSanctam/UnamWebPanel/blob/b8b30b7b0412fa2f817fb80df9b561291eddb186/UnamWebPanel/assets/php/session-header.php#L8

**Comment by VK-VZ** [comment]
Let's fix the mistakes please
[php-errors.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/13759506/php-errors.zip)

And please tell me how to close the directory from viewing?

**Comment by UnamSanctam** [comment]
Thank you for the errors, I don't get those on my end but I will fix that for those that do. If you have apache then you can block access to the logs by changing the https://github.com/UnamSanctam/UnamWebPanel/blob/master/UnamWebPanel/.htaccess file to:
```
DirectoryIndex index.php
<FilesMatch "unamwebpanel\.db">  
  Require all denied
</FilesMatch>
<Directory "__UNAM_LIB/Logs">
  Require all denied
</Directory>
Options -Indexes
```

**Comment by VK-VZ** [comment]
1.Can workers perform poorly due to these PHP errors?  Or maybe workers are not added?

2. The .htaccess file was changed, the panel stopped working altogether. Is everything correct there?
 
![Screenshot_20231223_194900_Chrome](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149912866/dd0cfae5-7568-436b-9c70-67fa83e19ba5)
My server Apache, PHP 7.4.3

**Comment by UnamSanctam** [comment]
1. No those warnings won't have any impact.
2. Try changing `"__UNAM_LIB/Logs"` to `"__UNAM_LIB\Logs"` and see if it works.

**Comment by VK-VZ** [comment]
![Screenshot_20231223_200634_Chrome](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149912866/e62f39fc-d2d3-43a7-8ec0-0f38339e4bb4)

The same

**Comment by UnamSanctam** [comment]
Your server might not allow the "Directory" directive then. You can alternatively reverse the .htaccess file to the previous one, and then create a new .htaccess file inside the `__UNAM_LIB\Logs` folder and enter this into it:
```
<Files "*">
  Require all denied
</Files>
```

**Comment by VK-VZ** [comment]
Thanks a lot.  It worked👍🙂

**Comment by UnamSanctam** [comment]
Great!
